client/lib/search-box.js

```diff
@@ -12,7 +12,7 @@ module.exports = function () {
   });
   searchinput.addEventListener('keyup', debounce(function (e) {
     var q = e.target.value;
-    if (q.length > 0) {
+    if (q.length > 0 && e.code !== 'Enter') {
```

Comment:
I've checked the value linked to ```e``` and it seems that 'Enter' is a right value. I'll check if it's specific to a certain browser and if it exists a better value